pylint/checkers/imports.py

```diff
@@ -697,21 +697,29 @@ def _record_import(
         self._imports_stack.append((node, importedname))
 
     @staticmethod
-    def _is_fallback_import(node, imports):
+    def _is_fallback_import(
+        node: ImportNode, imports: list[tuple[ImportNode, str]]
+    ) -> bool:
         imports = [import_node for (import_node, _) in imports]
         return any(astroid.are_exclusive(import_node, node) for import_node in imports)
 
-    def _check_imports_order(self, _module_node: nodes.Module):
+    def _check_imports_order(
+        self, _module_node: nodes.Module
+    ) -> tuple[
+        list[tuple[ImportNode, str]],
```

Comment:
Maybe we could create an alias for that one ? It's used ~~a lot~~ (after full review: only in this file, but it's still arguable imo).

Reply:
Yeah we could, but it is not too long imo. I like making typing explicit where possible because I don't need to look up what an alias refers to when I try and understand a piece of code.